docs/en/tutorial/tutorial_sql_1.md

```diff
@@ -86,96 +100,96 @@ hour(trans_time) as f_trans_hour,
-  - Lower bound time of OpenMLDB must be > = Upper bound time
-  - The number of lower bound entries of OpenMLDB must be < = The number of upper bound entries
+  - At present, OpenMLDB cannot support using the time after the current row as the upper or lower bounds, such as,`1d FOLLOWING`. In other words, we can only deal with the historical time window. This also basically meets most of the application scenarios of feature engineering.
+  - The lower bound time must be `>=` the upper bound time.
+  - The lower bound row must follow the upper bound row.
+
+
+#### Example
+
+For the transaction table T1 shown above, we define two `ROWS_RANGE` windows and two `ROWS` windows. The windows of each row are grouped by user ID (' uid ') and sorted by transaction time (' trans_time '). The following figure shows the result of grouping and sorting.
 
 ![img](images/table_t1.png)
 
-The following example shows that for the transaction table t1 shown above, we have two windows defined by time and two defined by number of rows. Note that the following window definitions are not a completed SQL. We will add aggregate functions later to complete runnable SQL.
+Note that the following window definitions are not completed SQL. We will add aggregate functions later to complete runnable SQL.
 
 - w1d: the window within the most recent day
-
+The window of the user's most recent day containing the rows from the current to the most recent day
 ```sql
--- The window of the user's most recent day containing the rows from the current to the most recent day
 window w1d as (PARTITION BY uid ORDER BY trans_time ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW)
 ```
 
-The `w1d` window shown in the above figure is for the row with `id=9`, and the `w1d` window contains three rows (`id=6`, `id=8`, `id=9`). These three rows fall within the time window [2022-02-07 12:00:00, 2022-02-08 12:00:00] .
-
-- w1d_10d: the windows from 1 day ago to the last 10 days
+The `w1d` window shown in the above figure is for the row which `id=9`, and the `w1d` window contains three rows (`id=6`, `id=8`, `id=9`). These three rows fall in the time window [2022-02-07 12:00:00, 2022-02-08 12:00:00] .
 
+- w1d_10d: the window from 1 day ago to the last 10 days
 ```sql
--- The windows from 1 day ago to the last 10 days
 window w1d_10d as (PARTITION BY uid ORDER BY trans_time ROWS_RANGE BETWEEN 10d PRECEDING AND 1d PRECEDING)
 ```
 
-The window `w1d_10d` for the row with `id=9` contains three rows, which are with `id=1`, `id=3` and `id=4`. These three rows fall within the time window of [2022-01-29 12:00:00, 2022-02-07 12:00:00]。
+The window `w1d_10d` for the row which `id=9` contains three rows, which are `id=1`, `id=3` and `id=4`. These three rows fall in the time window of [2022-01-29 12:00:00, 2022-02-07 12:00:00]。
```

Comment:
The window `w1d_10d` for the row which `id=9`
The window `w1d_10d` for the partition `id=9`